lib/components/Toastr/index.js

```diff
@@ -17,11 +17,13 @@ const TOAST_CONFIG = {
   transition: Slide,
   position: toast.POSITION.BOTTOM_CENTER,
   hideProgressBar: true,
-  closeButton: (
+  closeButton: ({ closeToast, ...props }) => (
     <Close
       size={16}
       color="#68737D"
       className="neeto-ui-toastr__close-button"
+      onClick={closeToast}
+      {...props}
     />
```

Comment:
This is to fix the warning "React does not recognize the `closeToast` prop on a DOM element"